src/extensions/boosts.py

```diff
@@ -0,0 +1,12 @@
+import interactions as discord
+import main
+
+class boosts(discord.Extension):
+    def __init__(self, client):
+        self.client: discord.Client = client
+
+    @discord.listen()
+    async def on_message(self, event: discord.events.MessageCreate):
+        if event.message.type == 8:
+            channel = await self.client.fetch_channel(627542044390457350)
```

Comment:
Instead of using the literal id, maybe create a dict in `main.py` called `config` or `ids`, with a key:value like:
```py
config = {
  'channel_ids': {
    'lobby': 627542044390457350
  }
}
```
The channel id can be imported and referenced in many files this way.